TF2 Rich Presence/localization.py

```diff
@@ -119,7 +118,7 @@ def detect_system_language(log: logger.Log):
 
     if not db['has_asked_language']:
         language_codes: dict = {read_localization_files()[lang]['code']: lang for lang in langs[1:]}
-        system_locale: str = locale.windows_locale[ctypes.windll.kernel32.GetUserDefaultUILanguage()]
+        system_locale: str = locale.getlocale()[0]
```

Comment:
is `GetUserDefaultUILanguage` more accurate than `getlocale` ?

Reply:
IDK I probably copied this line from somewhere